### ISSUE TITLE
error while runing score.sh in aishell

### ISSUE THREAD

**Issue by lfgogogo** [hirofumi0810/neural_sp#241]
error message:./score.sh:line 64: 841Segmentation fault (core dumped) CUDA_VISIBLE_DEVICES=${gpu} ${NEURALSP_ROOT}/neural_sp/bin/asr/eval.py......
When i ran score.sh,it crashed with giving exact error like the error message.
How to solve it?

**Comment by lfgogogo** [comment]
I quantize the model using pytorch 1.3,it seems to be successful.When i run score.sh using pytorch 1.3,it gives the error message above,maybe someting wrong happen in the dataload stage.Then i run score.sh using pytorch 1.0.1,it gives the following error message:
AttributeError:can't get attribute '_rebuild_qtensor' ,rise from neuralsp/bin/train_utils.py,the torch.load() in load_checkpoint().

**Comment by hirofumi0810** [comment]
@lfgogogo Quantization is one of the milestones in this framework. Could you please consider sending a PR?
Also, what kind of model did you try to evaluate?

**Comment by lfgogogo** [comment]
Of course.I am not sure about the kind you say. I quantize the model.epoch-25 and the model-avg10 using pytorch1.3 has the quantization module,when i use the quantized model to evaluate,it gives errors above.Besides,i also use package distiller v0.3.2 to quantize the model,this raises an error:Unexpected key(s) in state_dict.I will make a PR soon.

**Comment by lfgogogo** [comment]
I can't login in github on the pycharm,so i can't pull request now,sorry for that.
Here is how i quantize the model.
`import torch
from neural_sp.bin.args_asr import parse_args_eval
from distiller.quantization.range_linear import PostTrainLinearQuantizer
import distiller.apputils as apputils
from neural_sp.trainers.lr_scheduler import LRScheduler
from neural_sp.trainers.optimizer import set_optimizer

argv=[]
args, _,_= parse_args_eval(argv)
model = Speech2Text(args, 'modelpath')
load_checkpoint('modelpath', model)

#1、using pytorch1.3
quantized_model=torch.quantization.quantize_dynamic(model, {torch.nn.Linear}, dtype=torch.qint8)

quantized_model_path='a path'
optimizer = set_optimizer(model, args.optimizer, args.lr, args.weight_decay)
is_transformer = 'former' in args.enc_type or 'former' in args.dec_type
scheduler = LRScheduler(optimizer, args.lr,
                            decay_type=args.lr_decay_type,
                            decay_start_epoch=args.lr_decay_start_epoch,
                            decay_rate=args.lr_decay_rate,
                            decay_patient_n_epochs=args.lr_decay_patient_n_epochs,
                            early_stop_patient_n_epochs=args.early_stop_patient_n_epochs,
                            lower_better=args.metric not in ['accuracy', 'bleu'],
                            warmup_start_lr=args.warmup_start_lr,
                            warmup_n_steps=args.warmup_n_steps,
                            peak_lr=0.05 / (getattr(args, 'transformer_enc_d_model', 0) **
                                            0.5) if 'conformer' in args.enc_type else 1e6,
                            model_size=getattr(args, 'transformer_enc_d_model',
                                               getattr(args, 'transformer_dec_d_model', 0)),
                            factor=args.lr_factor,
                            noam=args.optimizer == 'noam',
                            save_checkpoints_topk=10 if is_transformer else 1)
scheduler.save_checkpoint(quantized_model, quantized_model_path, remove_old=False)
#model.module.state_dict()->model.state_dict()

#2、using distiller
model = model.cuda()
model.eval()

quantizer = PostTrainLinearQuantizer(model)

quantizer.prepare_model()

apputils.save_checkpoint(0, 'model', model, optimizer=None,
                         name='quantized_model_name',
                         dir='quantized_model_path')`
